src/server/handler.js

```diff
@@ -1,107 +1,122 @@
-const fs = require("fs");
-const path = require("path");
-const getData = require("../queries/getData");
-const postData = require("../queries/postData");
-const qs = require("querystring");
+const fs = require('fs');
+const path = require('path');
+const getData = require('../queries/getData');
+const postData = require('../queries/postData');
+const qs = require('querystring');
 
-const homeHandler = (request, response) => {
-	const filePath = path.join(__dirname, "..", "..", "public", "index.html");
+const cookie = require('cookie');
+const jwt = require('jsonwebtoken');
+
+const { parse } = require('url');
+
+const homeHandler = (page, request, response) => {
+	let filePath = path.join(__dirname, '..', '..', 'public', `${page}.html`);
+	if (page === 'profile') {
+		console.log('cookies: ', request.headers.cookie);
```

Comment:
These are good for development but shouldn't be committed.

Reply:
do you mean the console log or these 4 lines?!

Reply:
The console logs 😸